internal/pipeline/testdata/invalidconfig/.expected/config.yaml

```diff
@@ -0,0 +1,2 @@
+exitCode: 1
+network: false
```

Comment:
I suppose the default value for network is false?

Reply:
+1, it should be omitted for brevity.

Reply:
done